configure.ac

```diff
@@ -441,6 +441,22 @@ AC_TRY_LINK(
   AC_MSG_RESULT([no])
 ])
 
+AC_MSG_CHECKING([for libcap-devel support])
+AC_TRY_LINK(
+#include <sys/capability.h>
+,
+[
+  int cap_version = (int) _LINUX_CAPABILITY_VERSION;
+  cap_version = cap_version;
+],
+[
+  AC_MSG_RESULT([yes])
+  AC_DEFINE(HAVE_LIBCAP, 1, [Define to 1 if libcap-devel library is installed])
+],
+[
+  AC_MSG_RESULT([no])
+])
+
```

Comment:
It seems that following line in configure.ac should be updated instead of previous
```
AC_CHECK_HEADERS([sys/prctl.h sys/inotify.h sys/fanotify.h sys/capability.h])
```
Use HAVE_SYS_CAPABILITY_H instead of HAVE_LIBCAP

Reply:
agree.